closed/custom/Docs.gmk

```diff
@@ -25,11 +25,10 @@ JVMTI_HTML := $(TOPDIR)/closed/variant-server/gensrc/jvmtifiles/jvmti.html
 
 OPENJ9_BASE_URL := https://www.eclipse.org/openj9/
 OPENJ9_BUG_SUBMIT_URL := https://github.com/eclipse/openj9/issues
-OPENJ9_FULL_COMPANY_NAME := IBM Corp
 OPENJ9_JAVADOC_BOTTOM := \
-    <a href="$(OPENJ9_BASE_URL)">Link</a> to Eclipse OpenJ9 documentation.<br> \
+    <a href="$(OPENJ9_BASE_URL)">Eclipse OpenJ9 documentation.</a><br> \
     To raise a bug report or suggest an improvement create an <a href="$(OPENJ9_BUG_SUBMIT_URL)">Eclipse Openj9 issue.</a><br> \
-    Copyright &copy; 1998, $(COPYRIGHT_YEAR), $(OPENJ9_FULL_COMPANY_NAME). All rights reserved.
+    Copyright &copy; 1998, $(COPYRIGHT_YEAR), IBM Corp. and others
```

Comment:
I think this should end with a period. The two preceding lines in the footer do.

Reply:
Added a period.